modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/util/ExportImportTaskUtil.java

```diff
@@ -37,10 +37,36 @@ public static ExportTask postExportTask(
 					"/JSON", _getQueryString(parameters)),
 				Http.Method.POST));
 
-		String externalReferenceCode = exportTask.getExternalReferenceCode();
+		return _exportTask(
+			exportTask.getExternalReferenceCode(), expectedExecuteStatus);
```

Comment:
In general, method names should contain a verb. In this case, you can call this "_executeExportTask" or similar.